packages/squiggle-lang/src/rescript/Distributions/GenericDist/GenericDist.res

```diff
@@ -75,6 +75,9 @@ let toPointSet = (~xyPointLength, ~sampleCount, t): result<PointSetTypes.pointSe
   }
 }
 
+let toSparkline = (~xyPointLength: int, ~sampleCount: int, ~buckets: int = 20, t: t) : result<string, error> => 
```

Comment:
Considering asking you to pass an `~env` object instead of two different parameters, to be consistent with `DistributionOperation.run`

Reply:
I'm trying to be consistent with the function that's above this, toPointSet? If I was to change this I should probably also change toPointSet

Reply:
Ok I'm gonna assume @OAGr had reasons for `GenericDist.toPointSet` and `DistributionOperaion.run` having this difference in signatures.

Reply:
Yea; GenericDist.toPointSet is more low-level. Is meant to have more precise signatures, so this is good.

Reply:
That said, there's a different issue with this: if ``toPointSet`` is called, it should be passed into this function. This is because ``toPointSet`` could take a while to run, so we'll want to be tracking it in the calling function. See how other functions in this file are done.